### ISSUE TITLE
A release to include latest fixes?

### ISSUE THREAD

**Issue by reasonerjt** [docker/notary#1117]
Hi,

Will there be a new release to include the latest fixes such as the fixes for #1097 and #1107?

**Comment by endophage** [comment]
Yes. We're looking at releasing once this project is complete https://github.com/docker/notary/projects/4

Target is end of April.
